website/docs/registry/faq.mdx

```diff
@@ -120,3 +120,25 @@ If your GPG key or Github account has been compromised, please [reach out to us
 ## Can I regain ownership of my artifact after losing access to my GitHub account?
 
 For support, please [reach out to us via email](/terraform/registry#getting-help).
+
+## Documentation Markdown Support
+
+### Why is my documentation not rendered like it is in Github?
+
+We make every effort to support GitHub Markdown syntax, but there are some limitations. To ensure consistent rendering across platforms, we recommend using generic Markdown syntax whenever possible.
+
+### Why are referenced links not rendered?
```

Comment:
It feels a little weird to me to just have this one example of a specific case where rendering won't work. I do understand the intent is to add to this as more divergences from GitHub syntax come to our attention. But are there any other cases we are aware of right now that we could add to make this feel a bit less odd?